<launch>
    <arg name="eta" value="0.6"/>
    <arg name="Geta" value="12.0"/>
    <param name="namespace_init_count" value="1"/>

    <node pkg="map_provider" type="assigner.py" name="assigner" output="screen">
        <param name="map_topic" value="/map"/>
        <param name="global_frame" value="/map"/>
        <param name="info_radius" value="1"/> 
        <param name="info_multiplier" value="3.0"/> 
        <param name="hysteresis_radius" value="3.0"/> 
        <param name="hysteresis_gain" value="2.0"/> 
        <param name="frontiers_topic" value="/filtered_points"/> 
        <param name="n_robots" value="1"/>
        <param name="namespace" value=""/>
        <param name="delay_after_assignement" value="0.5"/>
        <param name="rate" value="100"/>
        <param name="robot_frame" value="base_footprint"/>
    </node>

</launch>
